dds/idl/be_global.cpp

```diff
@@ -275,8 +320,12 @@ void invalid_option(char * option)
 void
 BE_GlobalData::parse_args(long& i, char** av)
 {
-  static const char WB_EXPORT_MACRO[] = "--export=";
-  static const size_t SZ_WB_EXPORT_MACRO = sizeof(WB_EXPORT_MACRO) - 1;
+  // This flag is provided for CIAO compatibility
+  static const char EXPORT_FLAG[] = "--export=";
+  static const size_t EXPORT_FLAG_SIZE = sizeof(EXPORT_FLAG) - 1;
+
+  static const char DEFAULT_TS_MACRO[] = "--default_nested=";
```

Comment:
Should be `--default-nested` to be consistent with the GNU style options in tao_idl and update the usage message in `be_util.cpp`.

Reply:
I can change that, but because there is a value after that, then it's going to look different from all other options.